.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,14 @@
-Want to contribute? Great! Make sure you've checked all these boxes before creating your PR:
+Want to contribute? Great! Make sure you've read, understood and considered all
```

Comment:
One sentence per line.

Reply:
Ah, I leave this to @cschuet.

This is never displayed as Markdown, it is only visible as pure text to our contributors in the issue text. For the README.md, Christoph preferred the readability of block-text over the diffability of one line per sentence.

Reply:
The commit message box does not resize and displays the raw text and not the MD as @SirVer mentioned, so I would prefer block-text for better readability.